src/decoratedtextinput/README.md

```diff
@@ -0,0 +1,54 @@
+# @dojo/widgets/decoratedtextinput/DecoratedTextInput widget
+
+Dojo 2's `DecoratedTextInput` widget extends `TextInput` to provide optional addons before or after the input, similar to those in (bootstrap)[https://v4-alpha.getbootstrap.com/components/input-group/#basic-example].
+
+
+## Features
+
+- Allows any number of addons to be inserted before or after the input
+- Includes all TextInput functionality
+
+### Accessibility Features
+
+The `DecoratedTextInput` does not add any built-in accessibility features above what is available in `TextInput`. If this widget is used, it should be done with careful consideration of what instructional text is needed to convey the information visually implied by the addon or addons. This should be done in the label text, and optionally supplemented by custom descriptive text and `aria-describedby`.
+
+After careful consideration we decided not to automatically point the input's `aria-describedby` attribute to the `id` or `id`s of the addons, as Bootstrap does in their examples. In all the cases we examined, the text of the addon was insufficient to describe the input required of the user, and in many cases it was actually more confusing.
```

Comment:
SHOTS FIRED

Reply:
Hopefully the updated readme text isn't as full of bullets 😂